### DISCUSSION TITLE
After convert dxf to image, how can I know where is point (0, 0) in image?

### DISCUSSION THREAD

**Discussion by nlc2002** [mozman/ezdxf#912] [Issues] [answered]
I wanted to convert dxf file to image and do some image processing, and generate a new dxf file. But I don't know where (0,0) maps to in the image. So it's hard to map back to dxf file.

**Comment by mbway** [comment, marked as answer]
see https://github.com/mozman/ezdxf/discussions/550
you probably want the DXF (WCS) to image transform

**Reply by nlc2002** [reply]
Thank you very much, you answer solved my urgent need!
